### PR TITLE
chore(nova): migrated all useQueries and useMutations into headless

### PR DESCRIPTION
**Pull request by mostafakamar2308**

### CODE REVIEW

**Review by neuodev — CHANGES_REQUESTED** [review]

**Review by neuodev — CHANGES_REQUESTED** [review]
Great work @mostafakamar2308. Your changes are well executed and very clean! Thank you! Please take a look at my minor comments.

File: packages/headless/src/Invoices.ts
[the remark this answers is not in the document]
Comment (mostafakamar2308) [reply packages/headless/src/Invoices.ts:1]: Thank you 🚀

File: packages/headless/src/interviews.ts [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+  const findInterviews = useCallback(
+    async ({ pageParam }: { pageParam: number }) => {
+      if (!profile) return { list: [], total: 0 };
+      return atlas.interview.findInterviews(profile.id, {
+        page: pageParam,
+        size: 10,
+      });
+    },
+    [profile]
+  );
```
Comment (neuodev) [inline packages/headless/src/interviews.ts:51]:
```suggestion
  user?: number
): useFindInterviewsProps {
  const atlas = useAtlas();
  const findInterviews = useCallback(
    async ({ pageParam }: { pageParam: number }) => {
      if (!user) return { list: [], total: 0 };
      return atlas.interview.findInterviews(user, {
        page: pageParam,
        size: 10,
      });
    },
    [user]
  );
```

File: packages/headless/src/interviews.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  return usePaginationQuery(findInterviews, [QueryKey.FindInterviewsPaged]);
+}
+
+export function useFindInterviews(profile: IUser.Self | null) {
```
Comment (neuodev) [inline packages/headless/src/interviews.ts:55]: `useFindInterviews` is the same as `useFindPagedInterviews`. Please remove `useFindInterviews` and rename `useFindPagedInterviews` to `useFindInterviews`.

File: packages/headless/src/tutor.ts [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+
+  return useQuery({
+    queryFn: findTutorAcivityScores,
+    queryKey: [QueryKey.FindTutorActivity, id],
+    enabled: !!id,
+    retry: false,
+  });
+}
+
+export function useFindUnpackedTutorRoles({
```
Comment (neuodev) [inline packages/headless/src/tutor.ts:85]:
```suggestion
export function useFindUnpackedTutorRules({
```

**Review by neuodev — CHANGES_REQUESTED** [review]
Great work @mostafakamar2308! Please take a look at my minor comments! Thanks!

File: packages/headless/src/interviews.ts (hunk 2 above)
Comment (mostafakamar2308) [reply packages/headless/src/interviews.ts:55]: @neuodev 
`useFindInterviews` is used in `TutorOnboarding` page as it expects an array of interviews, while usePaged is used in `interviews` page.

or would you like me to also change the logic in `TutorOnboarding` page to expect paged results?

**Review by neuodev — APPROVED** [review]
Great work @mostafakamar2308! Thank you! The code looks clean!

**Pull request merged**